examples/src/main/java/kafka/examples/Consumer.java

```diff
@@ -91,9 +99,13 @@ public void run() {
                     // we can't recover from these exceptions
                     Utils.printErr(e.getMessage());
                     shutdown();
+                } catch (OffsetOutOfRangeException | NoOffsetForPartitionException e) {
+                    // invalid or no offset found without auto.reset.policy
+                    Utils.printOut("Invalid or no offset found, using latest");
+                    consumer.seekToEnd(emptyList());
```

Comment:
We should feed the partitions in the exception. That is:
`consumer.seekToEnd(e.partitions())`

Reply:
I think this is correct. The javadoc says: "If no partitions are provided, seek to the final offset for all of the currently assigned partitions."

Reply:
Yes, I agree it won't have different result in this demo case. But I think we should provide a good example here to let users know, in their client application, they should use `consumer.seekToEnd(e.partitions())` to seek for the affected partitions only, not all of the assigned partitions. Does that make sense?

Reply:
Yes, that makes sense. Thanks Luke.